rocblas headers not found

```
In file included from /home/acxz/vcs/git/github/rocm-arch/rocm-arch/rocm-validation-suite/src/ROCmValidationSuite-rocm-5.2.0/src/rvs_blas.cpp:25:
/home/acxz/vcs/git/github/rocm-arch/rocm-arch/rocm-validation-suite/src/ROCmValidationSuite-rocm-5.2.0/rvslib/../include/rvs_blas.h:30:10: fatal error: 'rocblas.h' file not found
#include "rocblas.h"
         ^~~~~~~~~~~
```

--- a/babel.so/CMakeLists.txt
+++ b/babel.so/CMakeLists.txt
@@ -112,6 +112,9 @@ add_compile_options(-DBUILD_VERSION_STRING="${BUILD_VERSION_STRING}")
 set(ROCBLAS_LIB "rocblas")
 set(HIP_HCC_LIB "amdhip64")
 
+#ROCBLAS VERSION CHECK FLAGS TO CHECK REORG VERSION 2.44.0
+add_compile_options(-DRVS_ROCBLAS_VERSION_FLAT=${RVS_ROCBLAS_VERSION_FLAT})
+
 # Determine Roc Runtime header files are accessible
 if(NOT EXISTS ${HIP_INC_DIR}/include/hip/hip_runtime.h)
   message("ERROR: ROC Runtime headers can't be found under specified path. Please set HIP_INC_DIR path. Current value is : " ${HIP_INC_DIR})
@@ -126,7 +129,7 @@ endif()
 # Determine Roc Runtime header files are accessible
 if(DEFINED RVS_ROCMSMI)
   if(NOT RVS_ROCMSMI EQUAL 1)
-    if(NOT EXISTS ${ROCBLAS_INC_DIR}/rocblas.h)
+    if(NOT EXISTS ${ROCBLAS_INC_DIR}/${ROCBLAS_MODULE_NM_PREFIX}rocblas.h)
     message("ERROR: rocBLAS headers can't be found under specified path. Please set ROCBLAS_INC_DIR path. Current value is : " ${ROCBLAS_INC_DIR})
     RETURN()
     endif()

--- a/mem.so/CMakeLists.txt
+++ b/mem.so/CMakeLists.txt
@@ -113,6 +113,9 @@ add_compile_options(-DBUILD_VERSION_STRING="${BUILD_VERSION_STRING}")
 set(ROCBLAS_LIB "rocblas")
 set(HIP_HCC_LIB "amdhip64")
 
+#ROCBLAS VERSION CHECK FLAGS TO CHECK REORG VERSION 2.44.0
+add_compile_options(-DRVS_ROCBLAS_VERSION_FLAT=${RVS_ROCBLAS_VERSION_FLAT})
+
 # Determine Roc Runtime header files are accessible
 if(NOT EXISTS ${HIP_INC_DIR}/include/hip/hip_runtime.h)
   message("ERROR: ROC Runtime headers can't be found under specified path. Please set HIP_INC_DIR path. Current value is : " ${HIP_INC_DIR})
@@ -127,7 +130,7 @@ endif()
 # Determine Roc Runtime header files are accessible
 if(DEFINED RVS_ROCMSMI)
   if(NOT RVS_ROCMSMI EQUAL 1)
-    if(NOT EXISTS ${ROCBLAS_INC_DIR}/rocblas.h)
+    if(NOT EXISTS ${ROCBLAS_INC_DIR}/${ROCBLAS_MODULE_NM_PREFIX}rocblas.h)
     message("ERROR: rocBLAS headers can't be found under specified path. Please set ROCBLAS_INC_DIR path. Current value is : " ${ROCBLAS_INC_DIR})
     RETURN()
     endif()

